umm-spec-lib/src/cmr/umm_spec/test/echo10_expected_conversion.clj

```diff
@@ -307,7 +307,6 @@
       (assoc :ContactGroups nil)
       (update-in [:ContactPersons] expected-echo10-contact-persons)
       (update-in [:ProcessingLevel] su/convert-empty-record-to-nil)
-      (update-in [:Distributions] echo10-expected-distributions)
```

Comment:
Looks like `echo10-expected-distributions` is a var - can that var be deleted now?